protoc-gen-grpc-gateway/descriptor/registry.go

```diff
@@ -425,6 +425,14 @@ func (r *Registry) GetUseJSONNamesForFields() bool {
 	return r.useJSONNamesForFields
 }
 
+// FieldName returns the proto or JSON field name for the field, based on UseJSONNamesForFields.
+func (r *Registry) FieldName(f *Field) string {
```

Comment:
drive-by refactor, since I was adding another couple places where I needed to resolve a field name.

Reply:
:car: :gun: